scripts/run_vc_test_suite.sh

```diff
@@ -31,7 +31,8 @@ export PATH=$PATH:`pwd`
 
 # get the suite
 rm -rf ${SUITE_DIR}
-git clone --depth=1 -q https://github.com/w3c/vc-test-suite suite
+# todo clone from https://github.com/w3c/vc-test-suite as soon as https://github.com/w3c/vc-test-suite/pull/102 is merged.
```

Comment:
It would be nice to simply patch example 16 signatures rather than using a fork of the entire repo.